src/custom-element.js

```diff
@@ -1520,6 +1542,11 @@ function createBaseCustomElementClass(win) {
      */
     toggleLoading_(state, opt_cleanup) {
       assertNotTemplate(this);
+      if (state &&
+          (this.layoutCount_ > 0 || this.signals_.get('render-start'))) {
```

Comment:
No. I had a "gotcha" with this as well. The old code uses `layoutCount_ > 0` everywhere. I simply moved it here as a more central place. Basically what happens is that `layoutCount_` is like an attempt - it's always incremented: failure or success. While `isFirstLayoutCompleted_` only set on success.